lib/matplotlib/figure.py

```diff
@@ -372,8 +372,16 @@ def _suplabels(self, t, info, **kwargs):
         manual_position = ('x' in kwargs or 'y' in kwargs)
         suplab = getattr(self, info['name'])
 
-        x = kwargs.pop('x', info['x0'])
-        y = kwargs.pop('y', info['y0'])
+        x = kwargs.pop('x', None)
+        y = kwargs.pop('y', None)
+        if x is None and y is None:
+            autopos = True
+        else:
+            autopos = False
```

Comment:
Isn't this just `not manual_position` from line 372?

Reply:
oops, yes.

Reply:
I kept the new way, and just removed the `manual_postion`.  The new way lets the user manually specify `None` (for whatever reason).

Reply:
But like `manual_position`, you could write it as one line:
```suggestion
        autopos = x is None and y is None
```

Reply:
yes, fixed!